src/py/flwr/simulation/app.py

```diff
@@ -75,6 +108,21 @@ def start_simulation(  # pylint: disable=too-many-arguments
         An empty dictionary can be used (ray_init_args={}) to prevent any
         arguments from being passed to ray.init.
     """
+    list_clients: List[str]
```

Comment:
How about calling this `cids`?
```suggestion
    cids: List[str]
```